src/Build.UnitTests/TestAssets/ExampleNetTask/TestNetTask/TestNetTask.csproj

```diff
@@ -1,4 +1,4 @@
-<Project>
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
need this change to get project props

Reply:
to have these props
DOTNET_HOST_PATH => resolves path to dotnet.exe
RuntimeIdentifierGraphPath => gives the path to the sdk version resolved for the given project.